### PR TITLE
RSS-K8: Adding sequence registry test to be able to test K8 changes

### PR DESCRIPTION
**Pull request by mabansal**
Adding shuffle manager test with server sequence registry by that we can test the K8 registry changes end to end with Shuffle Manager

### CODE REVIEW

**Review by vectorijk — APPROVED** [review]

**Review by mayurdb — APPROVED** [review]

**Comment by mayurdb** [comment]
Can you please paste the result of running these tests locally?
Looks like the build and test are running only after the merge

File: src/test/scala/org/apache/spark/shuffle/RssShuffleManagerWithServerSequenceRegistryTest.scala [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    testStreamServer.shutdown()
+  }
+
+  @Test
+  def runSparkApplication(): Unit = {
+    val conf = TestUtil.newSparkConfWithStandAloneRegistryServer(appId, "")
+    conf.set("spark.shuffle.rss.serviceRegistry.type", "serverSequence")
+    conf.set("spark.shuffle.rss.serverSequence.connectionString",
+      s"localhost:${testStreamServer.getShufflePort}")
+    conf.set("spark.shuffle.rss.serverSequence.serverId", testStreamServer.getServerId)
```
Comment (vectorijk) [inline src/test/scala/org/apache/spark/shuffle/RssShuffleManagerWithServerSequenceRegistryTest.scala:58]: maybe we could land PR #45 first and use `newSparkConfWithSequenceServerRegistryServer` method to initialize the test server.

Comment (mabansal) [reply src/test/scala/org/apache/spark/shuffle/RssShuffleManagerWithServerSequenceRegistryTest.scala:58]: > maybe we could land PR #45 first and use `newSparkConfWithSequenceServerRegistryServer` method to initialize the test server.

This has to happen this way as we need to create a dummy registry and then let client know the service registry to use the s tatic list for the servers in K8

**Comment by mabansal** [comment]
> Can you please paste the result of running these tests locally?
> Looks like the build and test are running only after the merge

[INFO]
[INFO] Results:
[INFO]
[INFO] Tests run: 245, Failures: 0, Errors: 0, Skipped: 0
[INFO]
[INFO]
[INFO] --- jacoco-maven-plugin:0.8.2:report (report) @ remote-shuffle-service ---
[INFO] Loading execution data file /Users/mabansal/github-rss/RemoteShuffleService/target/jacoco.exec
[INFO] Analyzed bundle 'remote-shuffle-service' with 240 classes
[INFO]
[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ remote-shuffle-service ---
[INFO] Building jar: /Users/mabansal/github-rss/RemoteShuffleService/target/remote-shuffle-service-0.0.9.jar
[INFO]
[INFO] --- maven-shade-plugin:3.1.1:shade (client) @ remote-shuffle-service ---

**Comment by mabansal** [comment]
Thanks Merging in

**Pull request merged**